### PR TITLE
Website: :bug: code example chip to same on first page

### PR DESCRIPTION
**Pull request by JulianNymark**
### Description

The bug with the code examples still happens, but only on the first page load, when there is no fragment selected (and the current path doesn't match any of the chip paths)

Re-using the existing conditional check for selected seems to fix it for the first page as well.

### CODE REVIEW

File: aksel.nav.no/website/components/sanity-modules/code-examples/CodeExamples.tsx
Comment (HalvorHaugan) [inline aksel.nav.no/website/components/sanity-modules/code-examples/CodeExamples.tsx:116]: Don't really need that const now
```suggestion
                    router.replace(`${router.asPath.split("#")[0]}#${id}`);
```

**Review by HalvorHaugan — APPROVED** [review]
Oops 😇 Should have thought about this 🙈

**Comment by JulianNymark** [comment]
forgot to apply the const dissolving before I hit the merge button 🙈 (I think it's fine though)

**Pull request merged**
